src/vm/ClrEtwAll.man

```diff
@@ -3281,6 +3330,17 @@
                            task="CLRPerfTrack"
                            symbol="ModuleRangeLoad" message="$(string.RuntimePublisher.ModuleRangeLoadEventMessage)"/>
 
+                    <!-- Assembly Binder events -->
+                    <event value="300" version="0" level="win:Informational"  template="AssemblyBindStart"
```

Comment:
>value="300 [](start = 27, length = 10)

This value seems like it is in the middle of the sequence. Should it be moved later?